docs/guides/using-the-network-shim.md

```diff
@@ -0,0 +1,193 @@
+    chromeAllowXSiteCookies(on, config)
+}
+```
+
+#### `./cypress/support/index.js`
+
+The `enableNetworkShim` setup accepts no arguments and in order to execute a cypress run in a DHIS2 platform app, the `enableAutoLogin` setup is required, which means the file should look like this:
+
+```js
+import { enableAutoLogin, enableNetworkShim } from '@dhis2/cypress-commands'
+
+enableAutoLogin()
+enableNetworkShim()
+```
+
+### Package scripts and dependencies
+
+#### `./package.json`
+
+The following `devDependencies` are required:
+
+```json
+"devDependencies": {
+    "@dhis2/cli-utils-cypress": "^#.#.#",
+    "@dhis2/cypress-commands": "^#.#.#",
+    "@dhis2/cypress-plugins": "^#.#.#",
+},
+```
+
+And it is recommended to add some convenience commands to the `scripts` section, for example:
+
+```json
+"scripts": {
+    "cy:start": "BROWSER=none yarn start",
+    "cy:open": "wait-on 'http://localhost:3000' && cypress open",
+    "cy:run": "wait-on 'http://localhost:3000' && cypress run",
+    "cy:capture": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture' --kill-others --success first",
+    "cy:stub": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub' --kill-others --success first",
+},
+```
+
+Some notes:
+
+1. These example scripts assume both `wait-on` and `concurrently` are installed available. If this is not the case, run:
+    ```bash
+    yarn add -D wait-on concurrently
+    ```
+1. Setting `BROWSER=none` before running `yarn start` simply means that no browser tab is going to be opened/activated once the app is ready.
+1. In the example above no `dhis2ApiVersion` is passed as a cypress env variable, even though the Network Shim relies on this. This works because that variable is declared in `./cypress.env.json` so that value is used. See the [cypress docs](https://docs.cypress.io/guides/guides/environment-variables) for more info.
+1. The example above only takes into account testing against a single backend version. Suppose a v37 version of DHIS2 core was available on `http://localhost:8080` and a v36 version on `http://localhost:8081`, the package scripts could look like this:
+    ```json
+    "scripts": {
+        "cy:start": "BROWSER=none yarn start",
+        "cy:open": "wait-on 'http://localhost:3000' && cypress open",
+        "cy:run": "wait-on 'http://localhost:3000' && cypress run",
+        "cy:capture-v37": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture' --kill-others --success first",
+        "cy:stub-v37": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub' --kill-others --success first",
+        "cy:capture-v36": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=capture,dhis2BaseUrl=http://localhost:8081,dhis2ApiVersion=36' --kill-others --success first",
+        "cy:stub-v36": "concurrently 'yarn cy:start' 'yarn cy:run --env networkMode=stub,dhis2BaseUrl=http://localhost:8081,dhis2ApiVersion=36' --kill-others --success first",
```

Comment:
```suggestion
        "cy:capture-v36": "concurrently 'yarn cy:run --env networkMode=capture,dhis2BaseUrl=http://localhost:8081,dhis2ApiVersion=36' 'yarn cy:start' --kill-others --success first",
        "cy:stub-v36": "concurrently 'yarn cy:run --env networkMode=stub,dhis2BaseUrl=http://localhost:8081,dhis2ApiVersion=36' 'yarn cy:start' --kill-others --success first",
```

See comment above regarding these commands with `--success first`

Reply:
I was wrong: `--success first` means that concurrently will exit with 0 or 1 based on the result of the first command that finishes, regardless of the order